VAMobile/src/translations/en/common.json

```diff
@@ -180,11 +180,11 @@
   "appealDetails.vba": "Veterans Benefits Administration",
   "appealDetails.vha": "Veterans Health Administration",
   "appealDetails.videoConference": "video conference",
-  "appealDetails.viewMoreDetails": "To review more details about your appeal, visit VA.gov: ",
+  "appealDetails.viewMoreDetails": "To review more details about your appeal, go to VA.gov.",
   "appealDetails.viewPastEvents": "Review past events",
   "appealDetails.viewYourAppeal": "Review your appeal {{tabName}}",
-  "appealDetails.visitVAGov": "Visit VA.gov",
-  "appealDetails.visitVAGovA11yHint": "Navigates to V-A .gov claims and appeals. This page will open in your device's browser",
+  "appealDetails.goToVAGov": "Go to VA.gov",
```

Comment:
this string can be combined with

"fileUpload.goToVaGov": "Go to VA.gov",

I don't have a preference on which one gets combined but one of them should be removed

Reply:
Makes sense, I consolidated them both to just `goToVAGov`